### DISCUSSION TITLE
Ability to specify AVPlayer UserAgent

### DISCUSSION THREAD

**Discussion by cbarriga** [fethica/FRadioPlayer#66] [Q&A] [answered]
Hi,

Is there currently a way to set a custom UserAgent to the AVPlayer?

Thanks!

**Comment by fethica** [comment, marked as answer]
Hey @cbarriga 

Yes, you can set it in the code: https://github.com/fethica/FRadioPlayer/issues/37

I'll probably expose a variable to set it in the config.

**Reply by cbarriga** [reply]
Awesome! Thank you, I should've looked more :) But yes, that would be great if it can be exposed, I'm trying not to modify the library if I can help it.

Best,
Celso

**Reply by fethica** [reply]
Hey @cbarriga, version 0.1.22 is out with HTTP header fields support 👍

https://github.com/fethica/FRadioPlayer/releases/tag/0.1.22

**Comment by cbarriga** [comment]
Hi,

Thank you! I’ll grab the changes.

Best,
Celso

**Comment by cbarriga** [comment]
It works great! Thanks @fethica 

Best,
Celso
